config/locales/en.yml

```diff
@@ -17,3 +17,13 @@ en:
 
   titles:
     application: Figgy
+  valkyrie:
+    institution:
+      name: 'Princeton University Library'
+      homepage_url: 'https://library.princeton.edu'
+    product_name: 'Digital PUL'
```

Comment:
I'm going to create a separate ticket for this, but I've been thinking that DPUL is the brand for the frontend, so Plum/Figgy/whatever should have a different brand.